stats/collector.go

```diff
@@ -206,6 +213,20 @@ func (s *BufferedCollector) CollectMetadata(accountMetadata *Metadata) {
 	}
 }
 
+func (s *BufferedCollector) collectHeartbeat(ctx context.Context) {
+	// consistent id for all heartbeats of this run
+	const idLength = 16
```

Comment:
there's already a process ID that is generated on startup and sent with all events (see sender.go 103), no need to create another one here.

Reply:
Done.